emscripten.py

```diff
@@ -1806,10 +1806,11 @@ def create_fp_accessors(metadata):
     accessors.append('''
 Module['%(full)s'] = function() {
   %(assert)s
-  // Use the wasm function itself, for the table.
+  // Use the original wasm function itself, for the table, from the main module.
   var func = Module['asm']['%(original)s'];
-  // If there is no wasm function, this may be a JS library function or
-  // something from another module.
+  // Try an original version from a side module.
+  if (!func) func = Module['_%(original)s'];
```

Comment:
But why do side module symbols have an `_`?  Oh I guess they don't but when we add them to "Module" we artificially at one maybe?   Maybe we should add them to 'asm' too without the "_"?   since they are "native code symbols" too?

Reply:
Yes, I think when we add stuff to module, the support.js code adds a `_` to everything.

I don't think we can add them to `asm` - asm is an exports object from a wasm file, it's not a normal JS object.

I do think this asymmetry is not great, though - we should probably not use `asm` here, and have the main module write its symbols to `Module` like side modules do. But as the usage of `asm` was pre-existing here, I wasn't sure it makes sense to try to do a large refactoring here. But maybe you know that code better and have an idea?